src/externalized/DefaultContentManager.cc

```diff
@@ -1303,6 +1315,35 @@ const BloodCatSpawnsModel* DefaultContentManager::getBloodCatSpawnsOfSector(uint
 	return NULL;
 }
 
+const std::vector<const CreatureLairModel*> DefaultContentManager::getCreatureLairs() const
```

Comment:
You probably wanted to return a const reference here, you're returning a cost copy.